simtbx/kokkos/kokkos_ext.cpp

```diff
@@ -130,6 +174,43 @@ namespace simtbx { namespace Kokkos {
              (arg_("detector")),
              "Modify pixels with noise on CPU. Unusual pattern, returns pixels directly instead of saving persistent")
         .def("show",&simtbx::Kokkos::exascale_api::show)
+        .add_property("diffuse",
+             make_getter(&simtbx::Kokkos::exascale_api::diffuse,rbv()),
+             make_setter(&simtbx::Kokkos::exascale_api::diffuse,dcp()),
+             "the diffuse parameters for the simulation.")
+        ;
+
+      class_<simtbx::Kokkos::diffuse_api>("diffuse_api",no_init )
+        .def(init<>())
+        .add_property("enable",
+             make_getter(&simtbx::Kokkos::diffuse_api::enable,rbv()),
+             make_setter(&simtbx::Kokkos::diffuse_api::enable,dcp()),
+             "whether or not to simulate diffuse.")
+        .add_property("stencil_size",
+             make_getter(&simtbx::Kokkos::diffuse_api::stencil_size,rbv()),
+             make_setter(&simtbx::Kokkos::diffuse_api::stencil_size,dcp()),
+             "")
+        .add_property("symmetize_diffuse",
```

Comment:
symmetize or symmetrize?
I think it should be consistent between python and C++

Reply:
@Trzs incoming commit fixes the typo.